test/unit/test_Ites.cc

```diff
@@ -27,10 +27,10 @@ class IteManagerTest: public ::testing::Test {
         const vec<PTRef> & getTopLevelItes() const { return iteDag.getTopLevelItes(); }
         IteToSwitchInternal(Logic &l, PTRef tr) : IteToSwitch(l, tr) {}
     };
-    LRALogic logic;
+    ArithLogic logic;
     SRef lrasort;
```

Comment:
Any idea why we have this `lrasort` member here? We can get it with  `logic.getSort_real`, no? It looks like there is code like that in one test. Can this member be removed?

Reply:
Isn't it better if it's in one place?